test/OpenTelemetry.Instrumentation.AspNetCore.Tests/RouteTests/README.net6.0.md

```diff
@@ -0,0 +1,550 @@
+# Test results for ASP.NET Core 6
+
+| Span http.route | Metric http.route | App | Test Name |
+| - | - | - | - |
+| :broken_heart: | :broken_heart: | ConventionalRouting | [Root path](#conventionalrouting-root-path) |
+| :broken_heart: | :broken_heart: | ConventionalRouting | [Non-default action with route parameter and query string](#conventionalrouting-non-default-action-with-route-parameter-and-query-string) |
+| :broken_heart: | :broken_heart: | ConventionalRouting | [Non-default action with query string](#conventionalrouting-non-default-action-with-query-string) |
+| :broken_heart: | :broken_heart: | ConventionalRouting | [Not Found (404)](#conventionalrouting-not-found-404) |
```

Comment:
Shouldn't this be included in all routing types?

Reply:
Yes, there are a number of test cases like this one that are applicable for all the test applications. The suite of tests are not currently exhaustive. My main focus in this PR is to get the test framework in place so we can expand on it later.